spec/numberformat.html

```diff
@@ -618,6 +618,22 @@
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-intl.numberformat.prototype.formattoparts">
+      <h1>Intl.NumberFormat.prototype.formatToParts ( _value_ )</h1>
+
+      <p>
+        When the *Intl.NumberFormat.prototype.formatToParts* is called with an optional argument _value_, the following steps are taken:
+      </p>
+
+      <emu-alg>
+        1. Let _nf_ be *this* value.
+        1. If Type(_nf_) is not Object, throw a *TypeError* exception.
+        1. If _nf_ does not have an [[initializedNumberFormat]] internal slot, throw a *TypeError* exception.
```

Comment:
one minor editorial detail, `initializedNumberFormat` => `InitializedNumberFormat` now that internal slots are camel case. I can probably take care of that directly.

Reply:
done https://github.com/tc39/ecma402/commit/675495b49c64bc62d1815106abd9df5afa134128

Reply:
Thanks, @caridy!